sky/backends/backend_utils.py

```diff
@@ -1355,8 +1384,8 @@ def check_network_connection():
     try:
         http.head(_TEST_IP, timeout=3)
     except requests.Timeout as e:
-        raise exceptions.NetworkError(
-            'Could not refresh the cluster. Network seems down.') from e
+        raise exceptions.NetworkError('Network seems down.') from e
```

Comment:
More detailed error message, `Failed to page _TEST_IP, network seems down.`

Reply:
Good idea!